randovania/games/dread/json_data/Burenia.txt

```diff
@@ -1110,8 +1110,24 @@ Extra - polygon: [[-900.0, 3700.0], [-5000.0, 3700.0], [-5000.0, -2115.0], [-900
       Trivial
   > Pickup (Energy Tank)
       All of the following:
-          Flash Shift and Morph Ball
-          Grapple Beam or Spider Magnet or Walljump (Beginner)
+          # You always need morph
+          Morph Ball
+          Any of the following:
+              # Either flash or grapple with a trivial wall jump, or space jump directly
+              Flash Shift or Grapple Beam or Space Jump
+              All of the following:
+                  # Speed boost into spider magnet surface or beginner walljump
+                  Speed Booster
```

Comment:
Oh, I'm pretty sure you have to use the save station to get this, so this also needs "Disabled Door Lock Randomizer"

Reply:
Not true, although this one also requires disabled door lock randomizer.
i think that the area near the flash gate is large enough that you could charge a short charge there though.

https://user-images.githubusercontent.com/30442287/184507361-37374695-1606-423a-861f-06b2faa894ee.mp4

.

Reply:
yeah this still needs the door lock thing

Reply:
im also currently labbing whether you can use spark conservation to bring a spark in all the way from the top left door, and it is... very dumb. if the slope near the missile was one block lower and aligned with the door, this would be so much easier...

Reply:
the spark requires flash shift anyways to actually bring it far enough, but i'm still curious at least...

Reply:
I swear I had the door lock thing on an earlier version. I must have accidentally dropped it because it's a hassle to add new rows above existing nested ones. 😅 The spark from the flash shift room would be a more complicated version of the other spark, so until there's sufficient door lock logic in place to account for different doors then it's redundant.